lib/test_queue/iterator.rb

```diff
@@ -64,6 +65,7 @@ def each
       end
     rescue Errno::ENOENT, Errno::ECONNRESET, Errno::ECONNREFUSED
     ensure
+      $0 = procline
```

Comment:
Good catch to change it here, but why back to the default? As long as we're doing this, maybe something like "Recording test statistics."?